### ISSUE TITLE
Questions about using the model after training

### ISSUE THREAD

**Issue by pskucherov** [thuml/Time-Series-Library#582]
Thank you very much for your work! I use it and everything works great! I have questions about further use of the model after training. 

Over time, new data appears on which it is necessary to train.
1. How to continue training an already trained model on a new chunk of data?
2. How to continue training on the same data after stopping? Training large models takes more than 40 hours. If the computer freezes, then everything starts from scratch.
3. To use the data for forecasting, it needs to be normalized. But it needs to be normalized relative to the trained data. How can I do this?

* I understand how to implement this. But I would like to discuss it with you first. Maybe there are already ready-made solutions?

**Comment by wuhaixu2016** [comment]
Hi, great questions. Unfortunately, TSLib does not have a solution for these questions. However, I think you can resolve them easily.

1. You need to finetune the well-trained model, where a smaller learning rate is needed.

2. Firstly, you need to save the model checkpoints during training (maybe every epoch). Then, you can reload the model for continued training.

3. Currently, TSLib includes two types of normalizations. One is in the dataloader (https://github.com/thuml/Time-Series-Library/blob/main/data_provider/data_loader.py#L65-L68 ) and the other is in the input window (https://github.com/thuml/Time-Series-Library/blob/main/models/Nonstationary_Transformer.py#L116-L120 ). If you want to deploy the well-trained model, I think you need to follow the same normalization process as model training. Note that remember to inverse the prediction.
